workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+import classNames from 'classnames';
+
+import {
+  Page,
+  useChangelogPages,
+  useDocumentationPageLink,
+} from '<src>/components';
+import { useBatchCommits } from '<src>/hooks/useBatchCommits';
+import { useSpacingStyles, useUtilityStyles } from '<src>/styles';
+import { formatTimeAgo } from '<src>/utils';
+
+export const ChangelogHistory: FC = () => {
+  const containerSpacing = useSpacingStyles({
+    size: 3,
+  });
+  const batchCommitSpacing = useSpacingStyles({});
```

Comment:
Looking at this, this feels like it's too rigid which will end up requiring _more_ code to get the css applied as we want it